### DISCUSSION TITLE
Welcome to clingraph Discussions!

### DISCUSSION THREAD

**Discussion by susuhahnml** [potassco/clingraph#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by tortinator** [comment]
Hola @susuhahnml , when would you like to start on the paper? Not that my brain is back to shape but at least it should get prepared :smile:

**Reply by susuhahnml** [reply]
@tortinator  The new version is almost ready. I just found an issue with `clasp` json output and there is also a stange issue in the deploy in windows that I am checking with Roland. This week should be out with all the new documentation to get started on paper writting.

**Reply by tortinator** [reply]
No worries, I just thought that this was a nice occasion to test the discussion feature 😇
